test/metabase/query_processor/test_util.clj

```diff
@@ -581,14 +582,10 @@
   "Impl for `with-report-timezone-id`."
   [timezone-id thunk]
   {:pre [((some-fn nil? string?) timezone-id)]}
-  ;; This will fail if the app DB isn't initialized yet. That's fine — there's no DBs to notify if the app DB isn't
-  ;; set up.
-  (try
-    (#'driver/notify-all-databases-updated)
-    (catch Throwable _))
-  (binding [qp.timezone/*report-timezone-id-override* (or timezone-id ::nil)]
-    (testing (format "\nreport timezone id = %s" timezone-id)
-      (thunk))))
+  (driver.tu/wrap-notify-all-databases-updated
+    (binding [qp.timezone/*report-timezone-id-override* (or timezone-id ::nil)]
+      (testing (format "\nreport timezone id = %s" (pr-str timezone-id))
```

Comment:
That `pr-str` was the only conflict. I've added it also to the backport. It should not hurt to have it also here and also we would avoid possible future conflicts.